assets/src/dashboard/app/views/templateDetails/content/detailsGallery/index.js

```diff
@@ -51,23 +76,38 @@ function DetailsGallery({
 }) {
   const { NextButton, PrevButton } = useMemo(() => {
     const Previous = (
-      <PaginationButton
-        rotateRight
+      <Button
+        type={BUTTON_TYPES.TERTIARY}
+        size={BUTTON_SIZES.SMALL}
+        variant={BUTTON_VARIANTS.CIRCLE}
         aria-label={__('View previous template', 'web-stories')}
-        onClick={() => switchToTemplateByOffset(-1)}
+        onClick={({ currentTarget }) => {
+          currentTarget.blur();
```

Comment:
This fixes an annoying issue where when you page left or right the button focus doesn't go away! Something to do with the buttons being memoized i think? If anyone has a better solution, I'm ready for it.